stdlib/log-api/src/main/java/org/ballerinalang/stdlib/log/AbstractLogFunction.java

```diff
@@ -108,6 +109,9 @@ public String get() {
     //TODO merge below and above methods(below one new bvm)
     protected String getPackagePath(Context ctx) {
         // TODO add API method a suitable way to get package path or does this simply returns "ballerina/log"?
-        return ctx.getStrand().currentFrame.callableUnitInfo.getPackageInfo().getPkgPath();
+        Strand strand = ctx.getStrand();
+        String pkgPath = strand.getStack()[strand.fp-1].callableUnitInfo.getPackageInfo().getPkgPath();
```

Comment:
Had a chat with Rajith regarding this. He recommended calling strand.peek(1) instead of this and to add a conditional check for it.

```java
if (strand.fp > 0) {
    pkgPath = strand.peek(1).callableUnitInfo.getPackageInfo().getPkgPath();
    pkgPath = pkgPath.split(":")[0];
else {
    pkgPath = strand.peek().callableUnitInfo.getPackageInfo().getPkgPath();
}